content/en/blog/_posts/2023-10-19-Gateway-API-GA.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: blog
+title: "Gateway API is now Generally Available (GA)"
+date: 2023-10-19T10:00:00-08:00
+slug: gateway-api-ga
+---
+
+***Authors:*** Shane Utt (Kong), Nick Young (Isovalent), Rob Scott (Google)
+
+We (the community developing [Gateway API][gwapi]) are incredibly pleased and
+proud to announce _general availability_ of Gateway API for use with Kubernetes
+clusters `v1.24` and newer!
+
+## Important Highlights
+
+Gateway API brings a lot of new networking possibilities to the table in your
+Kubernetes cluster. In this post we'll cover some of the most important
+highlights.
+
+For a complete list of changes for this specific release, please see our
+[v1.0.0 changelog entry][chlogv1].
+
+For a complete list of every feature for every release, or to review some of
+the releases leading up to `v1.0.0`, please see our [historical
+changelog][clog].
+
+### Version 1 APIs
+
+This marks the **first stable release** of Gateway API, where we are now
+publishing the following APIs as `v1` (Version 1):
+
+- [GatewayClass][gwc]
+- [Gateway][gw]
+- [HTTPRoute][hr]
+
+Reaching version 1 with these APIs means we future iterations on them will be
+handled in a backwards compatible manner.
```

Comment:
Might want to clarify how closely Gateway will / won't follow https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions and https://kubernetes.io/docs/reference/using-api/deprecation-policy/
(linking to a document on this, covering Gateway's specific policy, is fine).

Reply:
~we~ or reword to:

```
Reaching version 1 with these APIs will follow the [stable](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions) API policy.
```

Reply:
Good call. I just pushed a significant update that include a link to Gateway API's versioning policy.